worldengine/imex/__init__.py

```diff
@@ -12,24 +12,49 @@
 import sys
 
 
-def export(world, export_type, export_bpp, export_signed, export_normalize):
+def export(world, export_filetype, export_bpp, export_signed, path = 'seed_output', export_raw = False):
```

Comment:
I don't like this, I expect export_raw by default when working with DEM. When you scale it, you're warping the original values. This is fine for visualizing in images, but otherwise very bad. You are destroying your data set to match your two extremes.